web/src/AppAnime.css

```diff
@@ -0,0 +1,98 @@
+.anime {
+    order: 0;
+    flex: 0 1 auto;
+    width: 100px;
+    text-align: center;
+    color: #fff;
+    margin: 10px 15px 0px 20px;
+    font-size: 30px;
+    padding: 15px 15px 15px 15px;
+    border-radius: 20px;
+    background-color: #EFC743;
+  }
+
+  .User-anime {
+    margin: 20px 60px 20px 60px;
+    padding: 10px 10px 10px 10px;
+    color: #fff;
+    text-align: justify;
+    line-height: 30px;
+  }
+
+  .animeInProgress {
+    order: 0;
+    flex: 1 1 auto;
```

Comment:
A lot of things from `anime*` can be extracted into single, common class.

Reply:
Done :)